packages/app/public/index.html

```diff
@@ -4,15 +4,23 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="description" content="Janus Community Showcase" />
+    <meta
+      name="description"
+      content="<%= config.getOptionalString('app.title') ?? 'Janus Community Showcase' %>"
```

Comment:
I think we can use RHDH here since we sunset Janus:

```suggestion
      content="<%= config.getOptionalString('app.title') ?? 'RHDH' %>"
```

Maybe this will require additional changes in e2e tests.